lib/pipeline/addon/components/settings-git/template.hbs

```diff
@@ -43,161 +66,47 @@
           <a href="{{if provider.tls 'https' 'http'}}://{{provider.hostname}}" target="_blank" rel="nofollow noreferrer">{{provider.hostname}}</a>
         </div>
       {{/if}}
-      <div><b>{{t 'authPage.authenticated.header.clientId.text'}} </b> <span class="text-muted">{{provider.clientId}}</span></div>
 
-      <div class="row mt-30 text-center">
-        {{#if confirmDisable}}
-          {{#if disabling}}
-            <button class="btn bg-disabled" disabled="disabled">
-              <i class="icon icon-spinner icon-spin"></i> {{t 'generic.saving'}}
-            </button>
-          {{else}}
-          <button class="btn bg-primary" {{action "disable"}}>
-            <i class="icon icon-alert"></i> {{t 'pipelinesSetting.sureToDisableOAuth'}}
+      {{#if (eq provider.name 'bitbucketcloud')}}
+        <div>
+          <b>{{t 'authPage.authenticated.header.enterpriseHostName'}} </b> 
+          <a href="https://bitbucket.org" target="_blank" rel="nofollow noreferrer">bitbucket.org</a>
+        </div>
+      {{/if}}
+
+      {{#if provider.clientId}}
+        <div>
+          <b>{{t (if isBitbucket 'authPage.authenticated.header.key.text' 'authPage.authenticated.header.clientId.text')}} </b>
+          <span class="text-muted">{{provider.clientId}}</span>
+        </div>
+      {{/if}}
+    </section>
+
+    <div class="row mt-30 text-center">
```

Comment:
Can you use {{save-cancel}} component here?

Reply:
There is a `Confirm Disable` step, so I guess I can not you `{{save-cancel}}` here